pkg/apiserver/plugins.go

```diff
@@ -112,6 +116,7 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	clusteropenidconnectpreset.Register(plugins)
 	shootstatedeletionvalidator.Register(plugins)
 	customverbauthorizer.Register(plugins)
+	managedseedvalidator.Register(plugins)
```

Comment:
I guess both new plugins should be enabled by default, or? Can you add them to `DefaultOnPlugins`?

Reply:
Yes, it was "lost" during the rebase I did 2 hours ago, thanks for noticing.